proto/query.proto

```diff
@@ -32,20 +32,21 @@ message Query {
         message Ok {
             oneof ok {
                 Empty empty = 1;
-                ReadableConceptTreeStream readable_concept_tree_stream = 3;
+                ConceptDocumentStream concept_document_stream = 3;
```

Comment:
`ConceptTree` becomes `Document` and `readable_concept` has disappeared everywhere!